study-builder/tenants/testboston/pages/login.html

```diff
@@ -156,6 +156,31 @@
         showPasswordRecentText = true;
     }
 
+    if (mode === 'signup') {
+        realOptions.additionalSignUpFields = [
+            {
+                name: "first_name",
```

Comment:
Name of custom fields should exactly match these, i.e. `first_name` and `last_name`, so they will get properly picked up in the `register-user-in-pepper.js` rule above. @charlottedye do these name look about right to you in terms of how you'll use it for Prion?

Reply:
first_name and last_name are the fields Prion is using.